optaplanner-docs/src/main/docbook/en-US/Chapter-Use_cases_and_examples/Chapter-Use_cases_and_examples.xml

```diff
@@ -21,15 +21,15 @@
       <title>Examples Overview</title>
 
       <tgroup cols="5">
-        <colspec align="left" colnum="1" colwidth="2*"/>
+        <colspec align="left" colnum="1" colwidth="1*"/>
```

Comment:
Hi Vidya, did you also take a look at the effect of this in the community html_single and pdf outputs?
To get those do:

   cd optaplanner-docs
   mvn clean install
   ls target/docbook/publish/en-US/ (or something like that)

We need to find a setting that is both acceptable for community and product.